package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "webpack",
+  "version": "1.0.0",
+  "description": "> Js Group Project made by a group of two teams",
+  "private": true,
+  "scripts": {
+    "test": "jest --env=jsdom",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yayner2002/webpack.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/yayner2002/webpack/issues"
+  },
+  "homepage": "https://github.com/yayner2002/webpack#readme",
```

Comment:
- [ ] Kindly change the `url` and `homepage` to the accurate URL for this repository. Kindly implement this change. 👍